doc/GuideForCommitters.md

```diff
@@ -26,6 +26,21 @@ follow when reviewing and merging pull requests and issues.  It also provides
 a checklist of items that committers must ensure have been completed prior to
 merging a pull request.
 
+## Committers and Areas of Expertise
+
+The following is an up to date list of the current committers within the project and their respective areas of expertise. It is encouraged to use this list as a guideline to request reviews on your pull requests from committers who is knowledgable in the area.
+
+| Committer          | Username       | Area of Expertise                               |
+|--------------------|----------------|-------------------------------------------------|
+| Andrew Young       | @youngar       | [cmake](https://github.com/eclipse/omr/labels/cmake), [jitbuilder](https://github.com/eclipse/omr/labels/jitbuilder), [port](https://github.com/eclipse/omr/labels/port)                         |
+| Angela Lin         | @alin00        |                                                 |
+| Charlie Gracie     | @charliegracie | [cmake](https://github.com/eclipse/omr/labels/cmake), [gc](https://github.com/eclipse/omr/labels/gc), [port](https://github.com/eclipse/omr/labels/port), [thread](https://github.com/eclipse/omr/labels/thread), [jitbuilder](https://github.com/eclipse/omr/labels/jitbuilder), [build](https://github.com/eclipse/omr/labels/build%20%2F%20configure), [ci](https://github.com/eclipse/omr/labels/ci)  |
+| Daryl Maier        | @0xdaryl       | [compiler](https://github.com/eclipse/omr/labels/compiler), [tril](https://github.com/eclipse/omr/labels/tril), [port](https://github.com/eclipse/omr/labels/port)                            |
```

Comment:
I hope this is not offensive but I would not have added port to Daryl or Filip but if everyone else feels this is ok I do not object.

Reply:
No offense taken at all. This is exactly the kind of feedback I was looking for. Thanks! Fixed in f52e03d